common/src/main/java/com/wynntils/models/profession/ProfessionModel.java

```diff
@@ -66,21 +57,18 @@ public class ProfessionModel extends Model {
 
     private static final int GATHER_COOLDOWN_TIME = 60;
```

Comment:
Oh, bit late now but these timers aren't used anymore.

Reply:
I guess, but that's still true at least. So ig it can stay.